bundles/redhat8.6/packages.txt.gotmpl

```diff
@@ -0,0 +1,36 @@
+audit
+ca-certificates
+conntrack-tools
+chrony
+ebtables
+open-vm-tools
+python3-pip
+socat
+sysstat
+yum-utils
+yum-plugin-versionlock
+NetworkManager
+yum-utils
+cloud-init
+cloud-utils-growpart
+container-selinux
+kubectl-{{ .KubernetesVersion }}-0
+kubelet-{{ .KubernetesVersion }}-0
+kubeadm-{{ .KubernetesVersion }}-0
+conntrack
+ebtables
+ethtool
+iproute
+iptables
+socat
+cri-tools
+gcc
+make
+elfutils-libelf-devel
+libseccomp
+nfs-utils
+iproute-tc
+kernel-headers-4.18.0-372.87.1.el8_6
+kernel-devel-4.18.0-372.87.1.el8_6
```

Comment:
How come you didn't set the version for RHEL 8.4 but you did here?

Reply:
The rhel 8.4 AMI was using the latest AMI with the latest kernel version.

The version of the kernel in the Rhel 8.6 is using a previous version so pulling the latest kernel-headers doesn't work